VirtoCommerce.Storefront.Model/Cart/Demo/LineItem.cs

```diff
@@ -2,6 +2,6 @@ namespace VirtoCommerce.Storefront.Model.Cart
 {
     public partial class LineItem
     {
-        public string ConfiguredProductId { get; set; }
+        public string ConfiguredGropupId { get; set; }
```

Comment:
I think there is a typo:  it should be ConfiguredGroupId

Reply:
Thank you.